### DISCUSSION TITLE
dicomizer telnet Interface

### DISCUSSION THREAD

**Discussion by mafi85** [nroduit/Weasis#317] [General]
hello,

is there a some documentation about the weasis dicomizer available.
It's possible to use the telnet interface for it?

**Comment by nroduit** [comment]
Unfortunately, there is no documentation on the Dicomizer yet.

Yes, but only [one command](https://nroduit.github.io/en/basics/commands/#acquirepatient) is available for the telnet or at startup.

To populate some metadata (Patient/Study), it is possible to configure a worklist or provide an [XML file](https://groups.google.com/g/dcm4che/c/JWQz3T7pP8k/m/Erud8Ew0DQAJ) with the acquire:patient command.

**Reply by mafi85** [reply]
Thanks for your answer.
I will check out the Worklist and xml option.
